webapp/channels/src/components/drafts/drafts_link/drafts_link.tsx

```diff
@@ -51,8 +51,9 @@ function DraftsLink() {
     const draftCount = useSelector(getDraftsCount);
     const teamId = useSelector(getCurrentTeamId);
     const teamScheduledPostCount = useSelector((state: GlobalState) => getScheduledPostsByTeamCount(state, teamId, true));
+    const isScheduledPostEnabled = useSelector(isScheduledPostsEnabled);
 
-    const itemsExist = draftCount > 0 || teamScheduledPostCount > 0;
+    const itemsExist = isScheduledPostEnabled ? draftCount > 0 || teamScheduledPostCount > 0 : draftCount > 0;
```

Comment:
This ternary operation is becoming quite complex :sweat_smile:

I wonder if for readability sake we should do something like...

```
const hasDrafts = draftCount  > 0;
const hasScheduledPost = teamScheduledPostCount > 0;
const itemExist = hasDrafts || (isScheduledPostEnabled ? hasScheduledPost : false);
```
(or any variation of the above, I may have overdone it :sweat_smile: )

Reply:
Done.